backend/app/adapter/db/change_log.go

```diff
@@ -0,0 +1,101 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/short-d/short/app/adapter/db/table"
+
+	"github.com/short-d/short/app/entity"
+	"github.com/short-d/short/app/usecase/repository"
+)
+
+var _ repository.ChangeLog = (*ChangeLogSQL)(nil)
+
+// ChangeLogSQL accesses ChangeLog information in change_log table through SQL.
+type ChangeLogSQL struct {
+	db *sql.DB
+}
+
+// GetChangeByID finds Change from change_log table with given ID.
+func (c ChangeLogSQL) GetChangeByID(id string) (entity.Change, error) {
+	statement := fmt.Sprintf(`SELECT "%s","%s","%s","%s" FROM "%s" WHERE "%s"=$1;`,
+		table.ChangeLog.ColumnID,
+		table.ChangeLog.ColumnTitle,
+		table.ChangeLog.ColumnSummaryMarkdown,
+		table.ChangeLog.ColumnReleasedAt,
+		table.ChangeLog.TableName,
+		table.ChangeLog.ColumnID,
+	)
+
+	row := c.db.QueryRow(statement, id)
+	var change = entity.Change{}
+
+	err := row.Scan(&change.ID, &change.Title, &change.SummaryMarkdown, &change.ReleasedAt)
+	return change, err
+}
+
+// GetChangeLog retrieves full changelog from change_log table.
+func (c ChangeLogSQL) GetChangeLog() ([]entity.Change, error) {
```

Comment:
Are you sure it is safe? I mean, `change_log` table is assumed to be `huge` in order to hold all changes. 
And I don't when we need to get the whole dataset. Maybe we should add pagination or other search criteria?

Reply:
We had discussion on issue here: https://github.com/short-d/short/issues/566 and a new issue was created: https://github.com/short-d/short/issues/595, pagination will be added with its corresponding PR